src/components/Steps/__stories__/steps.stories.mdx

```diff
@@ -0,0 +1,130 @@
+import Steps from "../Steps";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import Icon from "../../Icon/Icon.jsx"
+import { NavigationChevronLeft, NavigationChevronRight } from "../../Icon/Icons"
+import Tipseen from "../../Tipseen/Tipseen.jsx"
+import TipseenWizard from "../../Tipseen/TipseenWizard";
+import { modifiers } from "./helpers.js";
```

Comment:
I think there is no need for this helper in the steps stories because the jump problem is not appear there

Reply:
Unfortunately, without it, the Tipseen would be jumping